nutils/function.py

```diff
@@ -2193,12 +2193,8 @@ def dot(__a: IntoArray, __b: IntoArray, axes: Optional[Union[int, Sequence[int]]
 
     Parameters
     ----------
-    arg : :class:`Array` or something that can be :meth:`~Array.cast` into one
-    axis : :class:`int`, a sequence of :class:`int`, or ``None``
```

Comment:
You've removed the description of the `axis` argument, but the argument does exist (under the name `axes`). Can you restore the description for `axis` and rename `axis` to `axes`? Thanks!